e2e-tests/playwright/tests/functional/channels/edit_file_attachments/edit_file_attachment.spec.ts

```diff
@@ -0,0 +1,372 @@
+        'mattermost.png',
+        'archive.zip',
+    ]);
+
+    const post = await channelPage.centerView.getLastPost();
+    await post.toBeVisible();
+    await post.toContainText(originalMessage);
+    await post.toContainText('sample_text_file.txt');
+    await post.toContainText('mattermost.png');
+    await post.toContainText('archive.zip');
+
+    await post.hover();
+    await post.postMenu.toBeVisible();
+    await post.postMenu.clickOnDotMenu(page);
+
+    await channelPage.postDotMenu.toBeVisible();
+    await channelPage.postDotMenu.editMenuItem.click();
+    await channelPage.centerView.postEdit.toBeVisible();
+    await channelPage.centerView.postEdit.removeFile('sample_text_file.txt');
+    await channelPage.centerView.postEdit.removeFile('mattermost.png');
+    await channelPage.centerView.postEdit.removeFile('archive.zip');
+    await channelPage.centerView.postEdit.sendMessage();
+
+    const updatedPost = await channelPage.centerView.getLastPost();
+    await updatedPost.toBeVisible();
+    await updatedPost.toContainText(originalMessage);
+    expect(updatedPost).not.toContain('archive.zip');
+    expect(updatedPost).not.toContain('mattermost.png');
+    expect(updatedPost).not.toContain('sample_text_file.txt');
+});
+
+test('MM-T5655_2 removing message content and files should delete the post', async ({pw}) => {
+    test.setTimeout(120000);
+
+    const originalMessage = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit';
+
+    const {user} = await pw.initSetup();
+    const {page} = await pw.testBrowser.login(user);
+    const channelPage = new pages.ChannelsPage(page);
+
+    await channelPage.goto();
+    await channelPage.toBeVisible();
+    await channelPage.centerView.postCreate.postMessage(originalMessage, ['sample_text_file.txt']);
+
+    const post = await channelPage.centerView.getLastPost();
+    await post.toBeVisible();
+    await post.toContainText(originalMessage);
+    await post.toContainText('sample_text_file.txt');
+
+    await post.hover();
+    await post.postMenu.toBeVisible();
+    await post.postMenu.dotMenuButton.click();
+
+    await channelPage.postDotMenu.toBeVisible();
+    await channelPage.postDotMenu.editMenuItem.click();
+    await channelPage.centerView.postEdit.toBeVisible();
+    await channelPage.centerView.postEdit.removeFile('sample_text_file.txt');
+    await channelPage.centerView.postEdit.writeMessage('');
+    await channelPage.centerView.postEdit.sendMessage();
+
+    await channelPage.centerView.postEdit.deleteConfirmationDialog.toBeVisible();
+    await channelPage.centerView.postEdit.deleteConfirmationDialog.confirmDeletion();
+    await channelPage.centerView.postEdit.deleteConfirmationDialog.notToBeVisible();
+
+    expect(channelPage).not.toContain(originalMessage);
+    expect(channelPage).not.toContain('sample_text_file.txt');
+});
+
+test('MM-T5656_1 should be able to restore previously edited post version that contains attachments', async ({pw}) => {
```

Comment:
On quick scan, I don't see the verification related to "The files are not visible to other users."

Could you help confirm that this matches the steps/expectations from the [written test case](https://github.com/mattermost/mattermost-test-management/blob/main/data/test-cases/channels/edited-post/MM-T5656.md)?